sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -72,6 +91,15 @@ def test_check_estimator():
     # check that fit does input validation
     msg = "TypeError not raised by fit"
     assert_raises_regex(AssertionError, msg, check_estimator, BaseBadClassifier)
+    # check that sample_weights in fit accepts pandas.Series type
+    try:
+        from pandas import Series
```

Comment:
This is fine. Maybe again raise a skip if there is no pandas. I feel we should do that whenever there is a pandas import in the tests but that goes beyond this issue.